tests/python_client/testcases/test_alias.py

```diff
@@ -397,6 +397,30 @@ def test_alias_called_by_utility_has_partition(self):
 
         assert res is True
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_enable_mmap_by_alias(self):
+        """
+        target: enable or disable mmap by alias
+        method: enable or disable mmap by alias
+        expected: successfully enable mmap
+        """
+        self._connect()
+        c_name = cf.gen_unique_str("collection")
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        alias_name = cf.gen_unique_str(prefix)
+        self.utility_wrap.create_alias(collection_w.name, alias_name)
+        collection_w.set_properties({'mmap.enabled': True})
+        collection_alias, _ = self.collection_wrap.init_collection(name=alias_name,
+                                                                   check_task=CheckTasks.check_collection_property,
+                                                                   check_items={exp_name: alias_name,
+                                                                                exp_schema: default_schema})
+        collection_w.set_properties({'mmap.enabled': True})
+        pro = collection_w.describe().get("properties")
```

Comment:
Here should use collection_alias.describe() to check the mmap is set successfully or not? And the line "collection_w.set_properties({'mmap.enabled': True})" should be removed if you want to test the combination with mmap and alias?
And could add the steps of using alias to set properties and check using the original collection?

Reply:
> Here should use collection_alias.describe() to check the mmap is set successfully or not? And the line "collection_w.set_properties({'mmap.enabled': True})" should be removed if you want to test the combination with mmap and alias? And could add the steps of using alias to set properties and check using the original collection?

Yes, I should have used an alias to set up mmap